content/rancher/v2.x/en/installation/ha/rancheronRKE/_index.md

```diff
@@ -0,0 +1,242 @@
+You need to create a DNS A record, pointing to the IP address of your RKE nodes. If the DNS A record is created, you can validate if it’s setup correctly by running nslookup rancher.yourdomain.com. It should return the IP addresses of your RKE nodes like in the example below.
+
+```
+$ nslookup rancher.yourdomain.com
+Server:         your_nameserver_ip
+Address:        your_nameserver_ip#53
+
+Non-authoritative answer:
+Name:   rancher.yourdomain.com
+Address: ip_of_node1
+Name:   rancher.yourdomain.com
+Address: ip_of_node2
+Name:   rancher.yourdomain.com
+Address: ip_of_node3
+```
+
+**Using RKE**
+
+- Download RKE template
+  RKE uses a .yml manifest file to install and configure your k8s cluster. There are 2 templates to choose from, depending on the SSL certificate you want to use.
+
+  Download one of following templates:
+
+  - Using Self Signed Certificate [3-node-certificate.yml](https://github.com/rancher/rancher/blob/master/rke-templates/3-node-certificate.yml)
+  - Using Certificate Signed By Recognized CA [3-node-certificate-recognizedca.yml](https://github.com/rancher/rancher/blob/master/rke-templates/3-node-certificate-recognizedca.yml)
+  - Rename template to `rancher-cluster.yml`.
+
+- Edit RKE template
+  - Edit `rancher-cluster.yml` template 
+    - configure nodes section setting following placeholders
+      - `<IP>`: The IP address or hostname of the node.
+      - `<USER>`: The username to use to setup a SSH connection to the node. If the user is not the root user, make sure the user has access to the Docker socket. This can be tested by logging in on the node as the configured user and run docker ps.
+      - `<SSHKEY_FILE>`: The path of the SSH private key file used to authenticate to the node.
+    - configure addons section setting following placeholders
+       - `<BASE64_CRT>`: the base64 encoded string of the Certificate file (usually called cert.pem or domain.crt)
+       - `<BASE64_KEY>`: the base64 encoded string of the Certificate Key file (usually called key.pem or domain.key)
+       - `<BASE64_CA>`: the base64 encoded string of the CA Certificate file (usually called ca.pem or ca.crt)
+       - `<FQDN>`: the FQDN chosen in Configure DNS.
+  - Backup RKE template
+
+- Up RKE cluster
+
+  All configuration is in place to run RKE. Be sure you have installed it at you workstation. 
+  - Open a web browser and navigate to our [RKE Releases page](https://github.com/rancher/rke/releases/latest) 
+  - Download the latest RKE binary for your platform, `rke_linux-amd64`, `rke_darwin-amd64` or `rke_windows-amd64.exe`.
+  - Rename file to `rke` and move it to some folder in your PATH.
+
+  - To deploy RKE cluster and rancher HA as addons, just run:
+
+  ```
+  rke up --config rancher-cluster.yml
+  ```
+
+  - RKE will generate a `kubectl` config file named `kube_config_rancher-cluster.yml`. Backup this file, it will be use to connect to new RKE k8s cluster and upgrade rancher. 
+
+
+**Using rancher cli (Comming soon)** 
```

Comment:
We should format this and the other subsections with markdown headers (`###`) instead of bold. That way, they will be added to toc and you can reference them easily with hyperlinks.

Reply:
I think web presentation doesn't support `###` md format.